go.mod

```
@@ -282,6 +282,7 @@ replace (
 	go.etcd.io/etcd/pkg/v3 => go.etcd.io/etcd/pkg/v3 v3.5.0
 	go.etcd.io/etcd/raft/v3 => go.etcd.io/etcd/raft/v3 v3.5.0
 	go.etcd.io/etcd/server/v3 => go.etcd.io/etcd/server/v3 v3.5.0
+	gopkg.in/square/go-jose.v2 v2.5.1 => gopkg.in/square/go-jose.v2 v2.2.2
```

```suggestion
	gopkg.in/square/go-jose.v2 => gopkg.in/square/go-jose.v2 v2.2.2
```
Okey, no need v2.5.1 here? :)

---

fixed